DigitalLearningSolutions.Web/Views/RegisterDelegateByCentre/PersonalInformation.cshtml

```diff
@@ -48,7 +48,7 @@
                      autocomplete="family-name"
                      css-class="nhsuk-u-width-one-half" />
 
-      <vc:text-input asp-for="Email"
+      <vc:text-input asp-for="PrimaryEmail"

```

Comment:
Does this ticket cover the RegisterDelegateByCentre journey as well? Should we change label of this to "Primary email" add the secondary email to this page too?

Reply:
If so we can probably remove the Alias from this journey as well since it's a deprecated column

Reply:
The ticket text is was unclear, Register Delegate by Centre is to be dealt with separately. 'Registering for a Centre directly' means using a query parameter to prefill the centre field. I'll check that still works however before when doing markups.